### PR TITLE
Feature/create searchbar component

### PR DESCRIPTION
**Pull request by Nitesh058Raj**
Created Search Bar component and deleted color-variable.css file, the variables are in globles.css file now. please test and give approve;

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.vscode
```
Comment (WinayGupta777) [inline .gitignore]: - Add EOF character.
- Always remember that code does not ends at last line of file.

File: frontend/src/components/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      backgroundColor: 'var(--main-light-secondary-2)',
+      '& fieldset': {
+        borderColor: 'var(--main-light-secondary)',
+      },
+      '&:hover fieldset': {
+        borderColor: 'var(--main-light-secondary)',
+        border: '1px solid'
+      },
+      '&.Mui-focused fieldset': {
+        borderColor: 'var(--main-light-secondary)',
```
Comment (WinayGupta777) [inline frontend/src/components/SearchBar.js]: - Reformat `color` property by taking reference from my branch: `feature/left-drawer`
- For example: 
- `backgroundColor: 'var(--main-light-secondary-2)'` will be replaced by `backgroundColor: COLORS.<vaiable_name>`

File: frontend/src/styles/globals.css [hunk 3]
```diff
@@ -1,3 +1,13 @@
+:root{
+  --black: #000;
+  --white: #fff;
+  --main-primary: #66d9e8;
+
+  --main-secondary: #0b7285;
+  --main-light-secondary: #0c8599;
+  --main-light-secondary-2: #1098ad;
+}
+
```
Comment (WinayGupta777) [inline frontend/src/styles/globals.css]: - Remove this color variable declarations from here and add to `.js` file.
- Reference from branch `feature/left-drawer`

**Comment by Nitesh058Raj** [comment]
Done the required changes and will send the new pull req

**Pull request closed without merging**